resource-aggregate/.plantuml

```diff
@@ -237,4 +237,57 @@ deactivate Gateway
 ResourceAggregate --> Bus : DeviceMetadataUpdated
 deactivate ResourceAggregate
 deactivate Server
+@enduml
```

Comment:
Please don't forget to update diagrams here:
https://plgd.dev/architecture/component-overview/#commands-and-events-overview

Reply:
https://github.com/plgd-dev/plgd.dev/pull/16